aws/aws.go

```diff
@@ -567,6 +567,32 @@ func deleteStacks(cfClients map[string]cfClient, rdsClients map[string]rdsClient
 		}
 	}
 
+	if ctx.DeleteOrphanResources {
+		for r := range elbClients {
+			output, err := elbClients[r].DescribeLoadBalancers(&elb.DescribeLoadBalancersInput{})
```

Comment:
This will delete all load balancers, not just the orphans. Those are logged here https://github.com/hortonworks/cloud-haunter/blob/0443d6d7c5497ff3f16396d5a16143e7f808ea33/aws/aws.go#L1624 as part of native stack assembly